


  
Object ID: https://biocomputeobject.org/BCO_022118/1.0  
Spec Version: https://w3id.org/ieee/ieee-2791-schema/2791object.json  
etag: 7d83a2c7a9b9597df8fdf71e9567311a4a2046f57121bb35874357b5043ed88f
# Provenance
  
Name: Employing a modified HiChIP protocol to interrogate RNA-associated chromatin regions  
Version: 1.0  
License: Attribution 4.0 International (CC BY 4.0)  
Created: 2020-10-24T17:14:10.212Z  
Modified: 2020-10-24T17:14:10.212Z  

### Contributors
  

|Name|Contribution|Affiliation|Email|ORCID|
| :---: | :---: | :---: | :---: | :---: |
|Evgenia Ulianova |createdBy|George Washington University|eulianova@gwmail.gwu.edu|N/A|
|Maxwell R. Mumbach|authoredBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|Jeffrey M. Granja|authoredBy|Program in Biophysics, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|Ryan A. Flynn|authoredBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|http://orcid.org/0000-0001-5013-0442|
|Caitlin M. Roake|authoredBy|Department of Medicine, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|Ansuman T. Satpathy|authoredBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|http://orcid.org/0000-0002-5167-537X|
|Adam J. Rubin|authoredBy|Program in Epithelial Biology, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|Yanyan Qi|authoredBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|Zhaozhao Jiang|authoredBy|Program in Innate Immunity, University of Massachusetts Medical School, Worcester, MA, USA|N/A|N/A|
|Shadi Shams|authoredBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|Bryan H. Louie|authoredBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|Jimmy K. Guo|authoredBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|David G. Gennert|authoredBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|M. Ryan Corces|authoredBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|Paul A. Khavari|authoredBy|Program in Epithelial Biology, Stanford University School of Medicine, Stanford, CA, USA|N/A|http://orcid.org/0000-0003-0098-4989|
|Maninjay K. Atianand|authoredBy|Department of Immunology, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA|N/A|https://www.nature.com/articles/s41592-019-0407-x|
|Steven E. Artandi|authoredBy|Department of Medicine, Stanford University School of Medicine, Stanford, CA, USA|N/A|N/A|
|Katherine A. Fitzgerald|authoredBy|Program in Innate Immunity, University of Massachusetts Medical School, Worcester, MA, USA|N/A|N/A|
|William J. Greenleaf|contributedBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|http://orcid.org/0000-0003-1409-3095|
|Howard Y. Chang|contributedBy|Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA, USA|N/A|http://orcid.org/0000-0002-9459-4393|
  

### Review
  

|Name|Contribution|Affiliation|Email|ORCID|Status|Comments|Date|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Evgenia Ulianova|['createdBy']|George Washington University|eulianova@gwmail.gwu.edu|https://orcid.org/0000-0002-4541-2848|unreviewed|N/A|N/A|

# Usability
  
BCO Object for interrogating Telomerase RNA Component (TERC) - chromatin interaction. Three libraries were assembled: HiChIRP TERC sample from human female B lymphoblastoid cell line (GM12878) as well as RNase (GM12878) and HeLa cell line TERC-knockout samples as negative controls. Libraries were generated by performing HiChIRP (https://doi.org/10.1038/s41592-019-0407-x) and Illumina paired-end sequencing on the samples. Pipeline steps are repeated for each sample.
# Description

## Keywords
  
TERC, telomerase, interaction, chromatin, RNA
## External References
  

|Name|Namespace|Access Time|IDs|
| :---: | :---: | :---: | :---: |
|ChemIdplus|chemId|2020-10-20T17:13:00.000Z|0081295285|
|Sequence Ontology|so|2020-10-20T17:13:00.000Z|SO:0000624, SO:0001643, SO:0001496|
|PubChem|pubchem|2020-10-20T17:13:00.000Z|1259345|
|Taxonomy Browser|taxonomy|2020-10-20T17:13:00.000Z| 9606|

## Platform
  
Linux/Ubuntu
## Pipeline Steps
  
***Step 1: HiC-Pro initial processing of fastq***  
Version: 2.7.8  
Description: These steps are performed in parallel: Align to hg19, remove duplicates, assign fragments to mboI restriction enzyme, filter for valid interactions, generate binned interaction matrices, and filtered reads convert to .hic files using hicpro2juicebox  
**Prerequisites**  

|Name|uri|Access time|
| :---: | :---: | :---: |
|Bowtie 2.3.5|https://sourceforge.net/projects/bowtie-bio/files/bowtie2/2.3.5/|2020-10-22T01:01:00.000Z|
|R|http://cran.mirrors.hoobly.com/|2020-10-22T01:01:00.000Z|
|Python|https://pypi.org/project/rpython/#files|2020-10-22T01:01:00.000Z|
|Juicer tools|https://github.com/aidenlab/juicer/wiki/Download|2020-10-22T02:33:00.000Z|
|Java|https://java.com/en/download/manual.jsp|2020-10-22T02:28:00.000Z|
  
**Input List**  

|Name|uri|Access Time|
| :---: | :---: | :---: |
|N/A|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX5337603&cmd=search&m=downloads&s=seq|2020-10-22T01:13:00.000Z|
|N/A|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX5337602&cmd=search&m=downloads&s=seq|2020-10-22T01:14:00.000Z|
|N/A|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4821793&cmd=search&m=downloads&s=seq|2020-10-22T01:15:00.000Z|
|N/A|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4821792&cmd=search&m=downloads&s=seq|2020-10-22T01:46:00.000Z|
|N/A|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4184847&cmd=search&m=downloads&s=seq|2020-10-22T01:47:00.000Z|
|N/A|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4184846&cmd=search&m=downloads&s=seq|2020-10-22T01:47:00.000Z|
  
**Output List**  

|Name|uri|Access Time|
| :---: | :---: | :---: |
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3591885|2020-10-22T01:51:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3591884|2020-10-22T01:52:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3424971|2020-10-22T01:52:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3424970|2020-10-22T01:52:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3181426|2020-10-22T01:53:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3181425|2020-10-22T01:53:00.000Z|
  
***Step 2: Juicer***  
Version: 1.22.01  
Description: Identify high-confidence loops with HiCCUPS  
**Prerequisites**  

|Name|uri|Access time|
| :---: | :---: | :---: |
|Juicer Tools|https://github.com/aidenlab/juicer/wiki/Download|2020-10-22T03:01:00.000Z|
|Java|https://java.com/en/download/manual.jsp|2020-10-22T03:02:00.000Z|
  
**Input List**  

|Name|uri|Access Time|
| :---: | :---: | :---: |
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3591885|2020-10-23T03:03:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3591884|2020-10-23T03:03:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3424971|2020-10-23T03:03:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3424970|2020-10-23T03:04:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3181426|2020-10-23T03:04:00.000Z|
|N/A|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3181425|2020-10-23T03:04:00.000Z|
  
**Output List**  

|Name|uri|Access Time|
| :---: | :---: | :---: |
|N/A|https://www.nature.com/articles/s41592-019-0407-x#Sec25|2020-10-22T03:14:00.000Z|

# Parametric Domain
  

|Parameter|Step|Value|
| :---: | :---: | :---: |

# IO Domain

## Input Subdomain
  

|Filename|URI|Access Time|SHA Checksum|
| :---: | :---: | :---: | :---: |
|sra_data.fastq.gz|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX5337603&cmd=search&m=downloads&s=seq|2020-10-23T16:56:00.000Z|N/A|
|sra_data.fastq.gz|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX5337602&cmd=search&m=downloads&s=seq|2020-10-23T16:56:00.000Z|N/A|
|sra_data.fastq.gz|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4821793&cmd=search&m=downloads&s=seq|2020-10-23T16:57:00.000Z|N/A|
|sra_data.fastq.gz|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4821792&cmd=search&m=downloads&s=seq|2020-10-23T16:57:00.000Z|N/A|
|sra_data.fastq.gz|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4184847&cmd=search&m=downloads&s=seq|N/A|N/A|
|sra_data.fastq.gz|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4184846&cmd=search&m=downloads&s=seq|2020-10-23T16:58:00.000Z|N/A|

## Output Subdomain
  

|Media type|Filename|URI|Access Time|SHA Checksum|
| :---: | :---: | :---: | :---: | :---: |
|xlsx|41592_2019_407_MOESM5_ESM.xlsx|https://www.nature.com/articles/s41592-019-0407-x#Sec25|2020-10-23T17:03:00.000Z|N/A|

# Execution Domain
  
Script Driver: shell
## Scripts:
  

|Filename|URI|Access Time|
| :---: | :---: | :---: |
|HiC-Pro|https://github.com/nservant/HiC-Pro/blob/master/bin/HiC-Pro|2020-10-22T03:21:00.000Z|
|hicpro2juicebox.sh|https://github.com/nservant/HiC-Pro/blob/master/bin/utils/hicpro2juicebox.sh|2020-10-23T03:50:00.000Z|

## Environment Variables

## Software Prerequisites
  

|Name|Version|URI|Access Time|SHA Checksum|
| :---: | :---: | :---: | :---: | :---: |
|Bowtie 2.3.5|2.3.5|https://sourceforge.net/projects/bowtie-bio/files/bowtie2/2.3.5/|2020-10-22T03:29:00.000Z|N/A|
|R|4.0.0|http://cran.mirrors.hoobly.com/|2020-10-22T03:30:00.000Z|N/A|
|Python|0.2.1|https://pypi.org/project/rpython/#files|2020-10-22T03:32:00.000Z|N/A|
|Juicer tools|1.22.01|https://github.com/aidenlab/juicer/wiki/Download|2020-10-22T03:32:00.000Z|N/A|

## External Data Endpoints
  

|Name|URL|
| :---: | :---: |
|fastq for HeLa TERCKO HiChIRP TERC biological replicate 2|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX5337603&cmd=search&m=downloads&s=seq|
|fastq for HeLa TERCKO HiChIRP TERC biological replicate 1|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX5337602&cmd=search&m=downloads&s=seq|
|fastq for GM12878 HiChIRP RNase TERC biological replicate 2|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4821793&cmd=search&m=downloads&s=seq|
|fastq for GM12878 HiChIRP RNase TERC biological replicate 1|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4821792&cmd=search&m=downloads&s=seq|
|fastq for GM12878 HiChIRP TERC biological replicate 2|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4184847&cmd=search&m=downloads&s=seq|
|GM12878 HiChIRP TERC biological replicate 1|https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?exp=SRX4184846&cmd=search&m=downloads&s=seq|
|processed reads for  HeLa TERCKO HiChIRP TERC biological replicate 2|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3591885|
|processed reads for  HeLa TERCKO HiChIRP TERC biological replicate 1|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3591884|
|processed reads for GM12878 HiChIRP RNase TERC biological replicate 2|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3424971|
|processed reads for GM12878 HiChIRP RNase TERC biological replicate 1|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3424970|
|processed reads for GM12878 HiChIRP TERC biological replicate 2|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3181426|
|processed reads for  GM12878 HiChIRP TERC biological replicate 1|https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3181425|
